src/user.rs

```diff
@@ -14,9 +16,20 @@ use utils;
 ///
 /// Impure.
 pub fn new_user<P: AsRef<Path>>(public_key_path: P, name: String) -> Result<(), Error> {
+    // Add user
     let pub_key = try!(utils::file_as_string(public_key_path));
     let project = try!(utils::read_protonfile(None::<P>));
     let new_project = try!(project.add_user(&name, &pub_key));
-    utils::write_protonfile(&new_project, None::<P>)
-}
+    try!(utils::write_protonfile(&new_project, None::<P>));
+
+    // Commit changes
+    let signature = Signature::now("Proton Lights", "proton@teslaworks.net").unwrap();
+    let msg = format!("Adding {} as new user", name);
+    let pf_path = Path::new("Protonfile.json").as_ref();
```

Comment:
Is there a reason you do an `as_ref()` here instead of just doing a `&` reference on line 31? Or is that not the same?